### PR TITLE
883468  - How to make Winforms PdfViewer to have enriched features like WPF PdfViewer.

### PR DESCRIPTION
**Pull request by Dhileep4290**
**Description:**
                  Created UG for the How to section by adding separate UG content for the Hosting of WPF Pdfviewer in Winforms
                   *  This code change is for the UG for the Hosting of WPF PdfViewer in Winforms in the How to section of Winforms UG
                   *  Added detailed descriptions, examples, and guidelines for the UG for the Hosting of WPF PdfViewer in Winforms with code example.
                   * Added a seaparate md file for the UG for the Hosting of WPF PdfViewer in Winforms that is added as separate section in the How to section in Winforms UG

### CODE REVIEW

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+layout: post
+title: Enhance WinForms PdfViewer with WPF Features | WinForms | Syncfusion
+description: Learn how to integrate the WPF PdfViewer in Windows Forms to access advanced features like annotations, form filling, and signatures.
+platform: windowsforms
+control: PDF Viewer
+documentation: ug
+---
+# How to Add WPF PdfViewer Features to WinForms PdfViewer
+
+Currently the Syncfusion Windows Forms PdfViewer lacks support for the features, such as adding annotations, form filling, signatures, stamps, sticky notes, etc., which are available in the WPF PdfViewer.But we can integrate the WPF PdfViewer into the Windows Forms(How to host PDF Viewer in windows forms application?) to utilize the features that are not in the Windows Forms PdfViewer but are available in the WPF PdfViewer. For that first create a simple Windows Forms application
```
Comment (RajaVigneshB) [inline WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:11]: Add the KB link this in the discussed content

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Add the below code in the Form1.cs class (code behind) So that the embedding of the WPF PdfViewer into the Windows Forms will become ease. Use the below code in the Form1.cs class
+
+{% tabs %}
+{% highlight c# %}
+
+using System.Windows.Forms;
+using System.Windows.Forms.Integration;
+
```
Comment (RajaVigneshB) [inline WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:20]: As we discussed, please modify the complete document

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md (hunk 1 above)
Comment (Dhileep4290) [reply WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:11]: Added the KB link

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md (hunk 2 above)
Comment (Dhileep4290) [reply WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:20]: Modified the content and added explanation for the code

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+control: PDF Viewer
+documentation: ug
+---
+# How to Add WPF PdfViewer Features to WinForms PdfViewer
+
+To leverage the advanced features offered by the WPF PdfViewer, such as adding annotations, form filling, signatures, stamps, sticky notes, and more, you can [integrate the WPF PdfViewer into the Windows]( https://support.syncfusion.com/kb/article/7882/how-to-host-pdf-viewer-in-windows-forms-application ) and take advantage of its extensive feature sets.
+
+we can  host a WPF PdfViewer control within a Windows Forms application using an ElementHost. By embedding the WPF PdfViewer, you can access advanced features available in the WPF control while maintaining a Windows Forms interface.
+Below is an overview of the key components and steps involved in the implementation: 
+1. **Adding the ElementHost Control:**
```
Comment (RajaVigneshB) [inline WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:15]: The format of the UG was not followed here. Why we are using Asterix for the heading here?

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

**Review by Vikassekar — CHANGES_REQUESTED** [review]
As we discuss do the changes

**Review by RajaVigneshB — APPROVED** [review]

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+using System.Windows;
+using System.Windows.Controls;
+
+namespace SampleWF
+{
+    /// <summary>
+    /// Interaction logic for PdfViewer.xaml
+    /// </summary>
+    public partial class PdfViewer : UserControl
```
Comment (Vikassekar) [inline WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:58]: as discussed in the feedback call change the class name as PdfViewerUserControl

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+            // Create the ElementHost control to host the WPF PdfViewer
+            elementHost = new ElementHost();
+            elementHost.AutoSize = true;
+            elementHost.Size = panel1.Size;
+            elementHost.Dock = DockStyle.Fill;
+			// Adds the ElementHost to the panel1 control
+            panel1.Controls.Add(elementHost);
+			
+            // Create the WPF PdfViewer control and assign it as a child of ElementHost
+            PdfViewer pdfViewer = new PdfViewer();
```
Comment (Vikassekar) [inline WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:104]: as discussed in the feedback call, change  PdfViewerUserControl for custom control

**Review by Vikassekar — CHANGES_REQUESTED** [review]

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md (hunk 4 above)
Comment (Dhileep4290) [reply WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md]: changed the class name

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md (hunk 5 above)
Comment (Dhileep4290) [reply WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md]: changed the custom control's name

**Review by Vikassekar — APPROVED** [review]
Approving this

File: WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+		// Set the form to maximized when it loads
+        private void Form1_Load(object sender, System.EventArgs e)
+        {
+            this.WindowState = FormWindowState.Maximized;
+        }
+    }
+}
+
+{%endhighlight%}
+{% endtabs %}
```
Comment (Vikassekar) [inline WindowsForms/PDF-Viewer/How-To/How-to-make-Winforms-PdfViewer-have-enriched-features-like-WPF-PdfViewer.md:123]: Can you add the sample link for this UG like in the KB below https://support.syncfusion.com/kb/article/7882/how-to-host-pdf-viewer-in-windows-forms-application

**Review by Vikassekar — CHANGES_REQUESTED** [review]

**Review by Vikassekar — APPROVED** [review]

**Pull request merged**